### ISSUE TITLE
mobileassetd spam running in simulator

### ISSUE THREAD

**Issue by eseidelGoogle** [flutter/flutter#6393]
I don't know if flutter is triggering this spam or if it's just spam we should filter?

FYI @danrubel @devoncarew

```
mobileassetd: mobileassetd - _localURLForAsset: [com.apple.accessibility.AccessibilityUIServer] _localURLForAsset[com.apple.MobileAsset.VoiceServicesVocalizerVoice]: unable to get local URL for asset: Error Domain=MobileAssetError Code=1 "(null)" [com_apple_MobileAsset_VoiceServicesVocalizerVoice/67ea7dd3dda1183a978f1dca63b1fb1003787c2b.zip]
mobileassetd: mobileassetd - _localURLForAsset: [com.apple.accessibility.AccessibilityUIServer] _localURLForAsset[com.apple.MobileAsset.VoiceServicesVocalizerVoice]: unable to get local URL for asset: Error Domain=MobileAssetError Code=1 "(null)" [com_apple_MobileAsset_VoiceServicesVocalizerVoice/2df1ef549ebf72711802444b0d5a00e03e396ed7.zip]
mobileassetd: mobileassetd - _localURLForAsset: [com.apple.accessibility.AccessibilityUIServer] _localURLForAsset[com.apple.MobileAsset.VoiceServicesVocalizerVoice]: unable to get local URL for asset: Error Domain=MobileAssetError Code=1 "(null)" [com_apple_MobileAsset_VoiceServicesVocalizerVoice/90a91e7a11a26b25ff4e2a48c70ea5a8b6538c33.zip]
mobileassetd: mobileassetd - _localURLForAsset: [com.apple.accessibility.AccessibilityUIServer] _localURLForAsset[com.apple.MobileAsset.VoiceServicesVocalizerVoice]: unable to get local URL for asset: Error Domain=MobileAssetError Code=1 "(null)" [com_apple_MobileAsset_VoiceServicesVocalizerVoice/f6b3c76676e2fbf2411861af4a67cc141537c5fe.zip]
```

**Comment by eseidelGoogle** [comment]
http://stackoverflow.com/questions/39715301/ios10-mobileassetd-unable-to-get-local-url-for-asset

**Comment by eseidelGoogle** [comment]
It may just be a first-run issue.

**Comment by Hixie** [comment]
I'm going to close this since we haven't considered a high priority and if anyone wants to fix it they can just do so so we don't need to track it independently.
